packages/ui/src/components/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import { ChevronDownIcon } from '@heroicons/react/24/solid';
+import type { Meta, StoryFn } from '@storybook/react';
+
+import { Badge } from './Badge';
+
+const Icon = ChevronDownIcon;
```

Comment:
Can we use another icon here? It makes the UI kinda confusing (looks like a dropdown)